.storybook/preview.js

```diff
@@ -17,10 +21,23 @@ const styles = {
   alignItems: 'center',
 }
 
+const store = configureStore({
+  appState: {},
+
+  localeMessages: {
+    current: en,
+    en: en,
+  },
+})
+
 const CenterDecorator = story => (
```

Comment:
Nit: we should probably rename this decorator, since it's no longer just centering